sbin/build.sh

```diff
@@ -459,6 +463,41 @@ executeTemplatedFile() {
     exit 2
   fi
 
+  # Restore exit behavior
+  set -eu
+}
+
+createOpenJDKFailureLogsArchive() {
+    echo "OpenJDK make failed, archiving make failed logs"
+    cd build/*
+
+    local adoptLogArchiveDir="AdoptOpenJDKLogsArchive"
+
+    # Create new folder for failure logs
+    rm -rf ${adoptLogArchiveDir}
```

Comment:
Removing the old archive first on purpose?

Reply:
yes, just to ensure there isn't any remnant from a dirty workspace